src/endpoint/devices.ts

```diff
@@ -5,24 +5,62 @@ import { Links, Status, SuccessStatusValue } from '../types'
 import { PresentationDevicePresentation } from './presentation'
 
 
-const HEADER_OVERRIDES = {Accept: 'application/vnd.smartthings+json;v=20170916'}
+const HEADER_OVERRIDES = { Accept: 'application/vnd.smartthings+json;v=20170916' }
 
 export interface CapabilityReference {
 	id: string
 	version?: number
 	status?: CapabilityStatus
 }
 
+export interface DeviceProfileReference {
+	id?: string
+}
+
+// Included for backwards compatibility (renamed to match API docs).
+// For new code, use DeviceProfileReference.
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface ProfileIdentifier extends DeviceProfileReference {}
+
+export interface DeviceCategory {
+	name: string
+	categoryType: 'manufacturer' | 'user'
+}
+
+export interface Restrictions {
+	tier: number // integer
+	historyRetentionTTLDays?: number // integer
+	visibleWhenRestricted?: boolean // default: false
```

Comment:
Do these inline comments (ex. `// default: false`) show up in Intellisense code navigation? It would helpful if this info was included there, but it may require them to be doc blocks instead.

Reply:
Yeah, I've always meant to make those doc blocks but have been kind of overwhelmed by the sheer number of them and an uneasy feeling about their accuracy and completeness so I've never really followed through on that.

I've gone through and made a few of these (hopefully the most useful ones) into doc blocks.